test_neoxargs_usage Unit Test is Failing

**Describe the bug**
The `test_neoxargs_usage` in `tests/neox_args/test_neoxargs_usage.py` is failing on `main`. We should aim to have all tests on `main` to pass consistently. This test fails consistently enough to make me believe that it is entirely broken rather than flaky.

**To Reproduce**
Steps to reproduce the behavior:
1. Install required dependencies including the dev dependencies
2. Run `pytest tests/neox_args/` in the rpeo root
3. Observe that the test fails.  
```
========================================================================= short test summary info =========================================================================
FAILED tests/neox_args/test_neoxargs_usage.py::test_neoxargs_usage - AssertionError: all arguments used in code defined in NeoXArgs
================================================================= 1 failed, 15 passed, 1 warning in 2.69s =================================================================
```

**Expected behavior**
I'd expect that the unit test would pass on `main`.

**Proposed solution**
It appears that the test is failing due to an edge case where the configuration is next to a bracket. 

![image](https://user-images.githubusercontent.com/17308542/177049420-579eb722-df9e-43e7-9837-5bd7ac3acdf9.png)

I was able to get the `test_neoxargs_usage` test to pass when the changed the lookup to `re.findall(r"(?<=args\.).{2,}?(?=[\s\n(){}+-/*;:,=,[,\]])", file_contents)`

**Screenshots**
![image](https://user-images.githubusercontent.com/17308542/177203229-6141b1b8-0036-4fa8-91a6-67de903dc1f3.png)

**Environment (please complete the following information):**
 - GPUs: 8x NVIDIA A40
- Configs: Default

**Additional context**
There are dozens of other tests that are failing. The tests in `tests/model` are failing for other reasons. Since the problem has a large scope, I'm separating them into different issues.

--- a/tests/neox_args/test_neoxargs_usage.py
+++ b/tests/neox_args/test_neoxargs_usage.py
@@ -51,7 +51,7 @@ def test_neoxargs_usage():
 
         # find args matches
         matches = list(
-            re.findall(r"(?<=args\.).{2,}?(?=[\s\n(){}+-/*;:,=])", file_contents)
+            re.findall(r"(?<=args\.).{2,}?(?=[\s\n(){}+-/*;:,=,[,\]])", file_contents)
         )
         if len(matches) == 0:
             continue

